waku/node/waku_node.nim

```diff
@@ -954,15 +954,19 @@ proc lightpushPublish*(
       peer: RemotePeerInfo,
   ): Future[WakuLightPushResult[void]] {.async, gcsafe.} =
     if not node.wakuLightpushClient.isNil():
-      debug "publishing message with lightpush",
+      info "publishing message with lightpush",
         pubsubTopic = pubsubTopic,
         contentTopic = message.contentTopic,
-        peer = peer.peerId
+        target_peer_id = peer.peerId,
+        msg_hash = pubsubTopic.computeMessageHash(message).to0xHex()
       return await node.wakuLightpushClient.publish(pubsubTopic, message, peer)
 
     if not node.wakuLightPush.isNil():
       debug "publishing message with self hosted lightpush",
```

Comment:
Is this supposed to be `debug`?

Because in the `"publishing message with lightpush"` case, it is being logged at `info` level.

Reply:
As per previous conversations we had in PM meeting, I will move back all `info` to `debug`